### ISSUE TITLE
Issue when trying extend a design from the monorepo

### ISSUE THREAD

**Issue by darigovresearch** [freesewing/freesewing#1737]
**What seems to be the problem?**

After running `yarn kickstart` from the monorepo fork for extending a pattern from the link below causes an error of incompatible modules

https://freesewing.dev/howtos/code/extend-pattern

**Are you a FreeSewing patron?**

 - [ ] Yes, I am :hugs:
 - [x] No, I am not :thinking:

**Additional context**
![image](https://user-images.githubusercontent.com/30328618/151251710-f8f32d31-b2d3-4e91-a29a-ce2fe15ffb88.png)

This was found when trying to make bob the bib with #1615 extending the tutorial pattern

Can anyone please advise?

**Comment by nicholasdower** [comment]
Hi @darigovresearch , I believe the issue is to do with the node version being used. This was also recently reported in #1708 and a note was added to the README in #1710. Though I now see that was (mistakenly?) reverted in a2a4e2a83826c1428b8a3a740ea69e41b169f9b0.

You should be able to get this working by following the instructions here:
https://freesewing.dev/tutorials/getting-started-linux/installing-nvm
https://freesewing.dev/tutorials/getting-started-linux/node-versions

TLDR:

Install NVM then run:
```shell
nvm install v14.18.3
nvm use v14.18.3
```

**Comment by nicholasdower** [comment]
FYI the README now mentions the required NodeJs version. Changed in #1742.

Going to close this as a dup of #1708. Please feel free to reopen.

**Comment by joostdecock** [comment]
@nicholasdower all READMEs are auto-generated. That's why the changes were overwritten (and will be overwritten again).

Edit `config/templates/readme.main.md` instead to update the main README.

The entire [templates folder](https://github.com/freesewing/freesewing/tree/develop/config/templates) should give you an idea of other things that are auto-generated (package.json, READMEs for packages, rollup config, tests, ...)

**Comment by joostdecock** [comment]
@darigovresearch Use [nvm](https://github.com/nvm-sh/nvm) to manage node versions on your system.
Then using the NodeJS version you need becomes as easy as:

```sh
nvm install lts/fermium
nvm use lts/fermium
```

Essentially, you need NodeJS 14.

**Comment by darigovresearch** [comment]
Thank you both for taking a look at this & your guidance on how to resolve it. Definitely a duplicate of #1708, JavaScript is not our specialty but we hope to get better at it and understand the intricacies of it through this project!

**Comment by nicholasdower** [comment]
> Thank you both for taking a look at this & your guidance on how to resolve it. Definitely a duplicate of #1708, JavaScript is not our specialty but we hope to get better at it and understand the intricacies of it through this project!

Same here! I hope you got it working. Please let me know if not.
